templates/index.html

```diff
@@ -22,103 +22,112 @@ <h2 class="p-heading--one">Say hello to your physical cloud</h2>
       </div>
       <ul class="p-inline-list">
         <li aria-label="Find out how to install MAAS" class="p-inline-list__item"><a href="/install" class="p-button--positive" itemprop="url" >Install MAAS</a></li>
-        <li class="p-inline-list__item"><a aria-label="Watch the MAAS video on YouTube" class="p-button--neutral" data-type="youtube" data-autoplay="true" href="https://www.youtube.com/embed/eHfaiKlUYsM" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Video', 'eventAction' : 'Play', 'eventLabel' : 'Watch the MAAS video' });">Watch the video</a></li>
+        <li class="p-inline-list__item"><a aria-label="Watch the MAAS video on YouTube" class="p-button--neutral" data-type="youtube" data-autoplay="true" href="https://www.youtube.com/embed/eHfaiKlUYsM" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Video', 'eventAction' : 'Play', 'eventLabel' : 'Watch the MAAS video' });">Watch the video <span class="p-button__icon"><img class="p-button__icon-img" src="{{ ASSET_SERVER_URL }}17e7ea51-play-icon.svg" width="18" /></span></a></li>
       </ul>
     </div>
-    <div class="col-6 p-hero u-hidden--small">
-      <img class="p-hero__image" src="{{ ASSET_SERVER_URL }}8ca2eab9-hero-ui-image.png" width="870" alt="MAAS screenshot showing Nodes" />
-    </div>
+    <img src="{{ ASSET_SERVER_URL }}8ca2eab9-hero-ui-image.png?w=400" srcset="{{ ASSET_SERVER_URL }}8ca2eab9-hero-ui-image.png?w=400 800w, {{ ASSET_SERVER_URL }}8ca2eab9-hero-ui-image.png?w=750 1600w" class="u-hidden--small u-image-position--bottom u-image-position--right" alt="MAAS screenshot showing Nodes" />
```

Comment:
When the srcset attribute has any image candidate string with a width descriptor, the sizes attribute must also be present.